acoustid/data/submission.py

```
@@ -285,8 +303,11 @@
         query = query.limit(limit)
     count = 0
     for submission in ingest_db.execute(query):
-        import_submission(ingest_db, app_db, fingerprint_db, index, submission._mapping)
-        count += 1
+        result = import_submission(
+            ingest_db, app_db, fingerprint_db, index, submission._mapping
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)